readme_source.md

```diff
@@ -45,6 +45,12 @@ Please consult with your company's system administrator for more information on
 
 **SSH Key-Based Authentiation**
 1. When creating a Keyfactor certificate store for the remote file orchestrator extension (see "Creating Certificate Stores" later in this README, you may supply either a user id and password for the certificate store credentials (directly or through one of Keyfactor Command's PAM integrations), or a user id and SSH private key.  Both PKCS#1 (BEGIN RSA PRIVATE KEY) and PKCS#8 (BEGIN PRIVATE KEY) formats are supported for the SSH private key.  If using the normal Keyfactor Command credentials dialog without PAM integration, just copy and paste the full SSH private key into the Password textbox.
+2. For clients with OpenSSH 8.8 or later installed on their target Linux server (the server containing the certificate stores to be managed), there is are two additional steps that need to be performed if authenticating by SSH private key.  This is due to OpenSSH no longer natively supporting RSA-SSH (SHA1) as a HostKeyAlgorithm.  The library that this integration uses to communicate with the target server does require this.  We are currently working on a modification to use the more advanced RSA-SHA2-256 algorithm, but until that is complete, the following is necessary:

```

Comment:
"There is are two additional steps that need to be performed" should probably be "There are two additional steps that need to be performed on the target server"

and

"The library that this integration uses to communicate with the target server does require this" could be "The SSH library that this integration uses (SSH.NET), does not currently support RSA-SSH2-256 and needs the RSA-SSH algorithm available on the target server"